docs/zh/latest/plugins/aws-lambda.md

```diff
@@ -0,0 +1,213 @@
+---
+title: aws-lambda
+keywords:
+  - APISIX
+  - Plugin
+  - AWS Lambda
+  - aws-lambda
+description: 本文介绍了关于 Apache APISIX aws-lambda 插件的基本信息。
+---
+
+<!--
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+-->
+
+## 描述
+
+`aws-lambda` 插件用于将 [AWS Lambda](https://aws.amazon.com/lambda/) 作为动态上游集成至 `APISIX` ，将所有特定 `URI` 的请求代理到 `AWS` 云。
+
+插件启用后会终止对已配置 `URI` 的请求，并代表客户端向 `AWS Lambda Gateway URI` 发起一个新的请求，其中包括配置的授权详细信息、请求头、正文（三个参数都是从原始请求中传递的）。然后将带有头信息、状态代码和正文的响应返回给使用 `APISIX` 发起请求的客户端。
+
+本插件支持通过 `AWS API key` 和 `AWS IAM secrets` 进行授权。
+
+## 属性
+
+| Name                 | Type    | Required | Default | Valid values | Description                                                  |
+| -------------------- | ------- | -------- | ------- | ------------ | ------------------------------------------------------------ |
+| function_uri         | string  | 是       |         |              | 触发 `lambda` 无服务器函数的 `AWS API Gateway` 端点。        |
+| authorization        | object  | 否       |         |              | 访问云函数的授权凭证。                                       |
+| authorization.apikey | string  | 否       |         |              | 生成的 `API` 密钥，用于授权对 `AWS` 网关端点的请求。         |
+| authorization.iam    | object  | 否       |         |              | 用于通过 `AWS v4` 请求签名执行的基于 AWS IAM 角色的授权。 请参阅 [IAM 授权方案](#IAM授权方案)。 |
+| timeout              | integer | 否       | 3000    | [100,...]    | 代理请求超时，以毫秒为单位。                                 |
+| ssl_verify           | boolean | 否       | true    | true/false   | 当设置为 `true` 时执行 `SSL` 验证。                          |
+| keepalive            | boolean | 否       | true    | true/false   | 当设置为 `true` 时，保持连接的活动状态以便重复使用。         |
+| keepalive_pool       | integer | 否       | 5       | [1,...]      | 在关闭该连接之前，可以在该连接上发送的最大请求数。           |
+| keepalive_timeout    | integer | 否       | 60000   | [1000,...]   | 在关闭该连接之前，可以在该连接上发送的最大请求数。           |
+
+### IAM 授权方案
+
+| Name       | Type   | Required | Default       | Description                                                  |
```

Comment:
Please translate, refer to [plugins/fault-injection.md](https://github.com/apache/apisix/blob/master/docs/zh/latest/plugins/fault-injection.md?plain=1).